test/test_quantized.py

```diff
@@ -169,6 +170,31 @@ def test_qrelu_leaky(self, X, alpha):
         self.assertEqual(qY.dequantize(), qY_hat.dequantize(),
                          message="F.leaky_relu failed ({} vs {})".format(qY, qY_hat))
 
+    """Tests the correctness of the quantized::gelu op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 5, 1, 5),
+                       qparams=hu.qparams()),
+           alpha=st.floats(0.0, 1.0, allow_nan=False, allow_infinity=False))
+    def test_qgelu(self, X, alpha):
+        X, (scale, zero_point, torch_type) = X
+
+        X = torch.from_numpy(X)
+        qX = torch.quantize_per_tensor(X, scale=scale, zero_point=zero_point,
+                                       dtype=torch_type)
+
+        # calculate GT
+        dqX = qX.dequantize()
+        # GELU
+        dqY_hat = dqX * 0.5 * (1.0 + (dqX / math.sqrt(2)).erf())
```

Comment:
Isn't it better to use the floating point version of the gelu here (`nn.functional.gelu`).